### PR TITLE
Added Mick's insignias

### PR DESCRIPTION
**Pull request by JoramD0**
**When merged this pull request will:**
- Add Mick's Minotaur insignias

### CODE REVIEW

File: addons/insignias/config.cpp [hunk 1]
```diff
@@ -8,7 +8,7 @@ class CfgPatches {
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"tacs_main"};
         author = ECSTRING(main,Author);
-        authors[] = {"Jonpas", "BadHabitz", "System98"};
+        authors[] = {"Jonpas", "BadHabitz", "System98","Mick Mathews"};
```
Comment (jonpas) [inline addons/insignias/config.cpp]: Space before his name, like before others.
Also did you ask him if he wants that name here or would rather prefer his nickname or something else?

**Comment by JoramD0** [comment]
Mick made some Helios patches aswell, he didn't know if you'd like them though.
HeliosPatchWhite
HeliosPatchBlack

File: addons/insignias/config.cpp (hunk 1 above)
Comment (JoramD0) [reply addons/insignias/config.cpp]: I asked him if he wanted his name or something else, he said his name

**Comment by SyMP2005** [comment]
I have to add these to the armory?

**Comment by JoramD0** [comment]
You shouldn't need to (assuming you are talking about the Olympus arsenal)

**Comment by SyMP2005** [comment]
We also have insignia available for purchase in the armory, which can then be applied with the armory on the contracts server. That's what I meant.

**Comment by JoramD0** [comment]
Ah, I guess that is up to you guys then. I do know you'd make Mick a happy 😉

**Comment by SyMP2005** [comment]
Always good to keep the oldies happy 😄

**Comment by jonpas** [comment]
Haha, yeah once this is released, which might take a while.

**Pull request merged**